sources/api/apiserver/src/server/mod.rs

```diff
@@ -121,6 +122,7 @@ where
             )
             .service(web::scope("/updates").route("/status", web::get().to(get_update_status)))
             .service(web::resource("/exec").route(web::get().to(exec::ws_exec)))
+            .service(web::scope("/report").route("", web::get().to(get_report)))
```

Comment:
I had expected to see a `/report/cis?level=2&format=json` or `/report/cis/2` instead of `/report?type=cis&level=2&format=json` and allow different types of reports to be routes within this API.

It's more complex, but then this would allow `/report/list` to list the available reports. I'm not sure if I'm just mistaken on how we have typically designed these APIs in the past to keep it consistent though.

Reply:
We could certainly do it that way. The only main downside I see is having a little more duplication of boilerplate code to route to the right processing (maybe?) versus the current argument approach. I could see a `/list` endpoint being useful for the `apiclient` discovery. I'll take a look at changing things.

Reply:
One small change - rather than `/report/list` to get the available reports, it's more "restful" to have the GET on `/report` to get its options, then `/report/cis` to get the specific report.